requirements.txt

```diff
@@ -1,8 +1,9 @@
-le-pycaption==2.2.0a1
+pycaption==2.0.7
+xblock-utils>=2.1,<3
 requests>=2.9.1,<3.0.0
 babelfish>=0.5.5,<0.6.0
-XBlock==1.4.0
+XBlock==1.4.0,<2
 django-statici18n==1.9.0
 
 -e git+https://github.com/edx/i18n-tools.git@v0.5.3#egg=edx-i18n-tools
-git+https://github.com/edx/xblock-utils.git@2.1.1#egg=xblock-utils==2.1.1
+# git+https://github.com/edx/xblock-utils.git@2.1.1#egg=xblock-utils==2.1.1
```

Comment:
? why do we leave commented requirement line?

Reply:
OMG, I forgot to remove it, thanks